src/Interactivity/directives/utils.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+require_once  __DIR__ . '/class-woo-directive-store.php';
+
+function store( $data ) {
```

Comment:
The function names in this file might later collide with the ones we're using in the Block Interactivity API. Have you considered prefixing them?